src/Validation/Validation.php

```diff
@@ -475,9 +477,12 @@ public static function decimal($check, $places = null, $regex = null)
         }
 
         // account for localized floats.
-        $data = localeconv();
-        $check = str_replace($data['thousands_sep'], '', $check);
-        $check = str_replace($data['decimal_point'], '.', $check);
+        $formatter = Number::formatter();
+        $decimalSeparator = $formatter->getSymbol(NumberFormatter::DECIMAL_SEPARATOR_SYMBOL);
+        $thousandsSeparator = $formatter->getSymbol(NumberFormatter::GROUPING_SEPARATOR_SYMBOL);
+        
+        $check = str_replace($thousandsSeparator, '', $check);
+        $check = str_replace($decimalSeparator, '.', $check);
```

Comment:
If we were going to use `NumberFormatter` for validation, then the `parse` method should be used instead of replacing the separators. This should also be made optional